### PR TITLE
The life of bailey the developer

### PR DESCRIPTION
**Pull request by bam95899**

### CODE REVIEW

File: sciencelabs/course/__init__.py [hunk 1]
```diff
@@ -13,5 +14,5 @@ def __init__(self):
         self.base = CourseController()
 
     def index(self):
-        course_info = self.base.get_course_info()
+        course_info = Course.get_course_info(self)
```
Comment (c-schwarze) [inline sciencelabs/course/__init__.py]: do you need to pass self in as a parameter? That seems weird

File: sciencelabs/db_repository/CourseProfessorsRepo.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from sqlalchemy import *
+
+from sciencelabs.db_repository import Base
+from sciencelabs.db_repository import session
+
+
+class CourseProfessors(Base):
+    __tablename__ = 'CourseProfessors'
+    course_id = Column(Integer, primary_key=True)
```
Comment (c-schwarze) [inline sciencelabs/db_repository/CourseProfessorsRepo.py]: It is probably worth looking into using foreign keys for these tables

File: sciencelabs/db_repository/CourseRepo.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    room = Column(String)
+
+    def get_course_info(self):
+        return (session.query(Course.title, Course.section, Course.dept, Course.course_num, User.firstName,
+                              User.lastName, Course.num_attendees).filter(Course.num_attendees)
+                .filter(User.id == CourseProfessors.professor_id).filter(CourseProfessors.course_id == Course.id)
+                .filter(Course.semester_id == Semester.id).filter(Semester.active == 1).all())
+
+    def get_report_course_info(self):
+        return (session.query(Course.dept, Course.course_num, Course.title, Course.section, User.firstName,
```
Comment (c-schwarze) [inline sciencelabs/db_repository/CourseRepo.py]: did we decide you can just do `session.query()` to get all the data?

File: sciencelabs/course/__init__.py (hunk 1 above)
Comment (c-schwarze) [reply sciencelabs/course/__init__.py]: also, i see this in a few other spots

File: sciencelabs/templates/course/base.html [hunk 4]
```diff
@@ -15,7 +15,7 @@
                             <p><strong>{{ app_settings['LAB_TITLE'] }} Courses for ***SEMESTER YEAR***</strong></p>
                         </div>
                         {# TODO FILL TABLE #}
-                        <table id="example" class="display nowrap" width="100%">
+                        <table id="course-table"  class="table table-striped table-bordered" style="width:100%">
```
Comment (c-schwarze) [inline sciencelabs/templates/course/base.html]: inline css should be removed

Comment (c-schwarze) [reply sciencelabs/templates/course/base.html]: I also see this on a few tables

File: sciencelabs/db_repository/CourseRepo.py (hunk 3 above)
Comment (bam95899) [reply sciencelabs/db_repository/CourseRepo.py]: Using session.query(**table model**) to get all data for whatever table

**Pull request merged**